js/main.js

```diff
@@ -0,0 +1,95 @@
+const libraryDiv = document.getElementById('Library');
+const errorElem = document.getElementById('errors');
+const btnNewBook = document.getElementById('btnNewBook');
+const btnAddBook = document.getElementById('btnAddBook');
+let myLibrary = [];
+
+function Book(title, author, desc) {
+  this.constructor = Book;
+  this.id = Math.random().toString(36).substr(2, 9);
+  this.title = title;
+  this.author = author;
+  this.desc = desc;
+  this.wasRead = false;
+}
+
+Book.prototype.toggleRead = function toogleRead() {
+  this.wasRead = !this.wasRead;
+};
+
+/* eslint-disable no-use-before-define */
+const removeBook = (e) => {
+  const { bookId } = e.target.dataset;
+  myLibrary = myLibrary.filter(book => book.id !== bookId);
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+  render();
+};
+
+const readBook = (e) => {
+  const { bookId } = e.target.dataset;
+  const book = myLibrary.find(book => book.id === bookId);
+  book.toggleRead();
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+  render();
+};
+/* eslint-enable no-use-before-define */
+
+const bookToHTML = (book) => `
+  <div class="book col-md-6">
+    <div class="card shadow">
+      <div class="card-body">
+        <h5 class="card-title">${book.title}</h5>
+        <h6 class="card-subtitle mb-2 text-muted">By ${book.author}</h6>
+        <p class="card-text">${book.desc}</p>
+        <button class="card-link btn btn-sm btn-read btn-${book.wasRead ? 'success' : 'secondary'}" data-book-id="${book.id}">
+          ${book.wasRead ? 'Mark as unread' : 'Mark as read'}
+        </button>
+        <button class="card-link btn btn-sm btn-danger btn-remove-book" data-book-id="${book.id}">Remove book</button>
+      </div>
+    </div>
+  </div>`;
+
+const errorToHTML = (error) => `
```

Comment:
Let separate the DOM from application logic, you can create another file for DOM.
![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/